src/mixins/_common.scss

```diff
@@ -0,0 +1,83 @@
+$expand-animation-timing: cubic-bezier(0, 0, 0.35, 1);
+
+// Grid auto fit sizes
+$grid-auto-fit-cell-width-small: 120px;
+$grid-auto-fit-cell-width-medium: 180px;
+$grid-auto-fit-cell-width-large: 240px;
+$grid-auto-fit-cell-width-xl: 300px;
+
+@mixin box-shadow-xs() {
+  box-shadow: var(--box-shadow-xs);
+}
+
+@mixin box-shadow-small() {
+  box-shadow: var(--box-shadow-small);
+}
+
+@mixin box-shadow-medium() {
+  box-shadow: var(--box-shadow-medium);
+}
+
+@mixin box-shadow-large() {
+  box-shadow: var(--box-shadow-large);
+}
```

Comment:
Looking at them makes me question if we need them at all... Maybe let's remove them, wdyt?

Reply:
Exactly what I thought, I agree